src/ext/property-panel/__tests__/data.test.ts

```diff
@@ -46,6 +49,54 @@ describe('data', () => {
       });
     });
 
+
+    describe('Unsupported Feature on DQ mode:', () => { 
+      let itemData: EngineAPI.IHyperCubeDimensionDef;
+      let args: {
+        properties: {
+          qHyperCubeDef: {
+            qIndentMode: boolean;
+          };
+        };
+      };
+
+      beforeEach(() => {
+        itemData = { qOtherTotalSpec: { qTotalMode: 'TOTAL_EXPR' } } as EngineAPI.IHyperCubeDimensionDef;
+        args = {
+          properties: {
+            qHyperCubeDef: {
+              qIndentMode: false,
+            },
+          },
+        } as Args;
+      });
+
+      afterAll(() => {
+        data = createData({
```

Comment:
There should be no need to do this. The beforeEach at :6 should already reset the data.